tests/robot-tests/tests/libs/slack.py

```diff
@@ -8,8 +8,26 @@
 
 PATH = f'{os.getcwd()}{os.sep}test-results'
 
+def _get_environment():
```

Comment:
Rather than get the environment this way, you could pass it as an argument to `send_slack_report`, as we have `args.env` in `run_tests.py`